src/Apps/Order/Routes/__tests__/Review.test.tsx

```diff
@@ -201,6 +229,30 @@ describe("Review", () => {
       expect(window.location.assign).toBeCalledWith("/artwork/artworkId")
     })
 
+    it("shows a modal with a helpful error message if the user's card is declined", async () => {
```

Comment:
Offer-mode orders will not try to charge the card when submitting, so it's OK to not have these two tests.